.github/workflows/azure-e2e.yaml

```diff
@@ -13,6 +13,8 @@ jobs:
       matrix:
         os:
         - "ubuntu 20.04"
+        - "ubuntu 18.04"
```

Comment:
```suggestion
```

Reply:
We support Ubuntu 18.04 for AKS. https://docs.d2iq.com/dkp/2.3/supported-infrastructure-operating-systems#id-(2.3)SupportedInfrastructureOperatingSystems-AKS 
However I notice that konvoy CLI does not use "gallery image" flag for aks cluster/nodepool create. 
so just to confirm, aks users will always use upstream azure image?

Reply:
Correct DKP users will use the default upstream images for AKS (and EKS).
I agree with @faiq we should only add configs that DKP supports.

Reply:
Thank you.